.travis.yml

```diff
@@ -43,14 +46,7 @@ install:
 
 
 script:
-  - if [ "${TRAVIS_OS_NAME}" = osx ]; then
-      target=all
-      GETTEXT="/usr/local/opt/gettext"
-      OPENSSL="/usr/local/opt/openssl";
-      export LIBRARY_PATH="$GETTEXT/lib"
-             CPATH="$OPENSSL/include:$GETTEXT/include"
-             PATH="$GETTEXT/bin:$PATH";
-    fi
+  - if [ "${TRAVIS_OS_NAME}" = osx ]; then target=all; fi
   - autoreconf -i &&
-    ./configure $CONF_FLAGS CFLAGS="$G_CFLAGS" &&
```

Comment:
Setting `CFLAGS` on the environment is not equivalent, check out the resulting Makefile.